src/components/WebexAudioSettings/WebexAudioSettings.stories.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
please remove this file, this component appears in storybook part of the WebexSettings component